zetta_utils/mazepa_layer_processing/annotation_postprocessing.py

```diff
@@ -0,0 +1,20 @@
+"""
+Module containing task and flow definitions for annotation layer operations.
+"""
+
+from zetta_utils import builder, mazepa
+
+# from zetta_utils.layer.volumetric import VolumetricAnnotationLayer
```

Comment:
Using `VolumetricAnnotationLayer` as a type annotation should work now, could you uncomment?